src/App.tsx

```diff
@@ -186,6 +194,9 @@ function App(props: RouteComponentProps) {
         setSelectedSvId('');
         leftReads.current = [];
         rightReads.current = [];
+
+        // Update the appearance of the clinical panel
+        setIsClinicalPanelOpen(!!demo?.clinicalInfo);
```

Comment:
Should we respect the previous state of whether the clinical panel is opened? If I understand correctly, I think you are force-opening the clinical panel when the user selects a new demo. I think this could be disturbing to the user.

```suggestion
        setIsClinicalPanelOpen(!!demo?.clinicalInfo && isClinicalPanelOpen);
```